sdk-api.yaml

```diff
@@ -26,27 +26,45 @@ servers:
   - url: http://localhost:8080/sdk-api
 
 tags:
-  - name: stage   # Generated class name is based on the tag; here, it's StageApi.
+  - name: stage # Generated class name is based on the tag; here, it's StageApi.
     description: "Stage API"
     externalDocs:
       description: "Stage API"
       url: https://docs.heystage.com/docs
+  - name: Organizations # Generated class name is based on the tag; here, it's StageApi.
+    description: "Organizations"
+    externalDocs:
+      description: "Stage API"
+      url: https://docs.heystage.com/docs
+  - name: Users # Generated class name is based on the tag; here, it's StageApi.
+    description: "Users"
+    externalDocs:
+      description: "Stage API"
+      url: https://docs.heystage.com/docs
+  - name: Billing # Generated class name is based on the tag; here, it's StageApi.
+    description: "Billing"
+    externalDocs:
+      description: "Stage API"
+      url: https://docs.heystage.com/docs
```

Comment:
Thank you for adding these @Stolkmeister . I'll have to check how these affect the generated SDK code. In the past, I had something similar but I had to remove because these affected the generated class names.

I'll check soon.

Reply:
I just tried generating the SDKs; we now have multiple different classes named after these tags. And, an end-point having different tags appears multiple times in the respective class. 

For example,  because`createOrganizationCheckoutSession` is tagged with `Billing` and `StageApi`, we have the following:

```
export class BillingApi extends BaseAPI {
  public createOrganizationCheckoutSession(organizationIdentifier: string, planIdentifier: string, checkoutSessionArg?: CheckoutSessionArg, options?: AxiosRequestConfig) {
      ...
    }
}

export class StageApi extends BaseAPI {
    public createUserCheckoutSession(userIdentifier: string, planIdentifier: string, checkoutSessionArg?: CheckoutSessionArg, options?: AxiosRequestConfig) {
      ...
    }
}
```

And so, `createUserCheckoutSession()` can be called via either of the following:

```
const stageApi = new StageApi(new Configuration({ ... }));
stageApi.createUserCheckoutSession();

const billingApi = new BillingApi(new Configuration({ ... }));
billingApi.createUserCheckoutSession();
```

This may be an improvement, but it seems like maybe something that we want to avoid (at least for now).

Reply:
This could have also been caused by how we generated the SDKs:

* For Axios:

```
openapi-generator generate -i sdk-api.yaml -g typescript-axios -o ./stage-typescript --additional-properties=npmName=stage-typescript,supportsES6=true
```

* For Fetch:

```
openapi-generator generate -i sdk-api.yaml -g typescript-fetch -o ./stage-typescript-fetch --additional-properties=npmName=stage-typescript-fetch,supportsES6=true,useSingleRequestParameter=false
```

Reply:
What do you think @kaiserstage and @les-stage ?

Reply:
I agree that duplicate tags might be confusing for some of your users. I added them because gives a bit more structure in the rendering of the specs. I can remove the tags with little impact for Alphadoc.

Reply:
> What do you think @kaiserstage and @les-stage ?

@dsidharta I agree, it is preferable and less confusing to not duplicate functions in multiple classes 👍